auth-web/src/util/config-helper.ts

```diff
@@ -152,4 +152,7 @@ export default class ConfigHelper {
     sessionStorage.setItem('BCREG-emailAddress', nameRequest.applicantEmail)
     sessionStorage.setItem('BCREG-phoneNumber', nameRequest.applicantPhone)
   }
+  static getAccountApprovalSlaInDays () {
+    return ConfigHelper.getValue('APPROVE_ACCOUNT_SLA_DAYS')
```

Comment:
just default to 5 incase if the value is empty or null ; like an ||5 to this condition will do